api/src/pcapi/settings.py

```diff
@@ -107,6 +107,7 @@
 USE_FAST_AND_INSECURE_PASSWORD_HASHING_ALGORITHM = bool(
     int(os.environ.get("USE_FAST_AND_INSECURE_PASSWORD_HASHING_ALGORITHM", False))
 )
+ENABLE_USER_CREATION_OR_UPDATE = bool(int(os.environ.get("SEND_TRANSACTIONNAL_EMAIL", 1)))
```

Comment:
Attention, erreur sur la variable d'environnement !
Question nommage, j'aurais préféré être plus clair, car on peut avoir l'impression que cette variable peut désactiver l'inscription d'utilisateurs.
```suggestion
ENABLE_IMPORT_TEST_USERS = bool(int(os.environ.get("ENABLE_IMPORT_TEST_USERS", 1)))
```